pkg/configuration/configuration.go

```diff
@@ -25,6 +25,8 @@ var (
 	BuildTime = "0"
 	// StartTime in ISO 8601 (UTC) format.
 	StartTime = time.Now().UTC().Format("2006-01-02T15:04:05Z")
+
+	prodEnvironment = "prod"
```

Comment:
this should be a const, not a var

Reply:
Done in https://github.com/codeready-toolchain/registration-service/pull/224/commits/f98ba62e3e1b53a95e8a7db698f66ea23cd7af9b